src/styles/BookList.css

```diff
@@ -25,12 +27,13 @@
 }
 
 .list-item {
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 0 15px 10px rgba(238, 239, 243, 0.8);
   display: flex;
```

Comment:
- [ ] Kindly use the correct color on the status circle. The remaining area on the circle is gray in the design. Please do the same thing to match the design. Kindly use the correct font family which is `Montserrat`. I suggest you minimize the size of the circle, it is looking a bit bigger when compared to the design. Please use the correct color for the text too, it is looking darker on your design. 🥸️

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/224475642-17dceb96-2911-47db-8fb2-6354621a1dd2.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/224475641-92ecd023-deea-496a-a405-e933bb0c5fb9.png" width="300">|